content/id/certificates.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Chain of Trust
+slug: certificates
+top_graphic: 5
+aliases: [/certs]
```

Comment:
Please remove the aliases line. It creates a redirect, and if there are multiple aliases from the same source path, the selection of which one redirects is random.

```suggestion
```